pkg/controller/podgroup_test.go

```diff
@@ -0,0 +1,181 @@
+package controller
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/kubernetes/pkg/controller"
+	st "k8s.io/kubernetes/pkg/scheduler/testing"
+
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/podgroup/v1alpha1"
+	pgfake "sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned/fake"
+	pgformers "sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions"
+	"sigs.k8s.io/scheduler-plugins/pkg/util"
+)
+
+func Test_Run(t *testing.T) {
+	ctx := context.TODO()
+	podSuccess := v1.PodSucceeded
+	podFailed := v1.PodFailed
+	createTime := metav1.Time{Time: time.Now().Add(-72 * time.Hour)}
+	cases := []struct {
+		name               string
+		pgName             string
+		minMember          uint32
+		podNames           []string
+		podNextPhase       *v1.PodPhase
```

Comment:
Can you make the type consistent? Make them all pointers, or all concrete types.